### DISCUSSION TITLE
Collection based on Sonarr/Radarr attributes/tags

### DISCUSSION THREAD

**Discussion by edrock200** [meisnate12/Plex-Meta-Manager#217] [Ideas]
Just a thought. I'm trying to find a way to make collections based on library path. I think the easiest way to do so would be to add a sonarr/radarr tag, or be able to pull the sonarr/radarr path attribute, to said items.

The only issue is I don't see a way of building collections from Sonarr/Radarr tags, unless I'm missing it. Just an idea for more possibilities.
Thanks for making this awesome tool!
-Ed

**Comment by meisnate12** [comment]
File path is exposed to plex so I shouldn't need radarr or sonarr for anything in that terms. But this currently isn't a feature it could be added tho

**Comment by meisnate12** [comment]
I added `filepath` and `filepath.not` as a collection filter in develop if you wanna try it out

**Comment by edrock200** [comment]
Thanks!

**Comment by miguelamaral** [comment]
Hi! Hijacking this to request a similar collection filter to search for substrings in filenames. Something like this:

```python
elif method_name == "filename":
    jailbreak = False
    for location in current.locations:
        for location_substring in filter_data:
            filename = os.path.basename(location)
            if location_substring.lower() in filename.lower():
                jailbreak = True
                break
        if jailbreak: break
    if (jailbreak and modifier == ".not") or (not jailbreak and modifier != ".not"):
        match = False
        break
```

This is working here, but I'm not sure if it'll also handle Windows paths, etc. The `lower()` case insensitive comparison might be too strict, but tbh it doesn't really make much of a difference in my use case.

Thanks in advance!

**Reply by meisnate12** [reply]
ok change is in develop it makes sense its just more inclusive
